pkg/virt-handler/vm.go

```diff
@@ -2974,18 +3051,26 @@ func (d *VirtualMachineController) claimDeviceOwnership(virtLauncherRootMount *s
 	return diskutils.DefaultOwnershipManager.SetFileOwnership(devicePath)
 }
 
-func (d *VirtualMachineController) reportDedicatedCPUSetForMigratingVMI(vmi *v1.VirtualMachineInstance) error {
+func getCpuSetArray(vmi *v1.VirtualMachineInstance) (cpusList []int, err error) {
 	cgroupManager, err := cgroup.NewManagerFromVM(vmi)
 	if err != nil {
-		return err
+		return []int{}, err
 	}
 
 	cpusetStr, err := cgroupManager.GetCpuSet()
 	if err != nil {
-		return err
+		return []int{}, err
 	}
 
 	cpuSet, err := hardware.ParseCPUSetLine(cpusetStr, 50000)
```

Comment:
you can return directly from `hardware.ParseCPUSetLine(cpusetStr, 50000)`

```
return hardware.ParseCPUSetLine(cpusetStr, 50000)
```

Reply:
Fixed, thanks.